Tests/FunctionalTests/PluginTests.swift

```diff
@@ -471,6 +471,28 @@ class PluginTests: XCTestCase {
         }
     }
 
+    func testLocalAndRemoteToolDependencies() throws {
+        // Only run the test if the environment in which we're running actually supports Swift concurrency (which the plugin APIs require).
+        try XCTSkipIf(!UserToolchain.default.supportsSwiftConcurrency(), "skipping because test environment doesn't support concurrency")
+
+        try fixture(name: "Miscellaneous/Plugins/PluginUsingLocalAndRemoteTool") { path in
```

Comment:
fixture already prints the error when caught so the catch here is redundant afaik

Reply:
Great, thanks — this was based on an older test so there might need to be more cleanup needed here.  I'll fix this one new test for now at least.

Reply:
Fixed it.